src/Microsoft.ML.FastTree/TreeEnsemble/InternalQuantileRegressionTree.cs

```diff
@@ -50,7 +50,7 @@ public InternalQuantileRegressionTree(byte[] buffer, ref int position)
             _instanceWeights = buffer.ToDoubleArray(ref position);
         }
 
-        public override void Save(ModelSaveContext ctx)
+        internal override void Save(ModelSaveContext ctx)
```

Comment:
>internal [](start = 8, length = 8)

Similar note that methods on internal classes don't need to themselves be marked internal.